doc/source/index.rst

```diff
@@ -7,5 +7,6 @@
    style-actions/index
    tests-actions/index
    doc-actions/index
+   check-licenses-action/index
```

Comment:
See previous comment about renaming this folder to unify the directories names.

Reply:
I have renamed the doc section to Licenses actions. However, I left the action itself as check-licenses. Let me know if you're OK with that @jorgepiloto.

Reply:
Yeah that sounds fine, don't worry. One thing is the action itself, something different are the docs. The action name should be as explicit and standalone as possible